orchestrator/schedules/validate_subscriptions.py

```diff
@@ -53,3 +54,8 @@ def validate_subscriptions() -> None:
             break
 
         start_validation_workflow_for_workflows(subscription=subscription, workflows=validation_product_workflows)
+
+
+@workflow("Validate subscriptions workflow", target=Target.SYSTEM)
+def validate_subscriptions_workflow() -> StepList:
```

Comment:
Would be better named `task_validate_subscriptions`, following the (loose) naming pattern of prefixing `task_` in the name of tasks